Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterInfo.tsx

```diff
@@ -160,8 +164,8 @@ const MeterInforWindow = (props: IProps) => {
                     {!valid('Name') ? <p> <ErrorSymbol /> Name must be less than 200 characters and is required.</p> : null}
                     {!valid('ShortName') ? <p> <ErrorSymbol />ShortName must be less than 50 characters.</p> : null}
                     {!valid('Alias') ? <p> <ErrorSymbol />Alias must be less than 200 characters.</p> : null}
-                    {!valid('Make') ? <p> <ErrorSymbol />Make must be less than 200 characters.</p> : null}
-                    {!valid('Model') ? <p> <ErrorSymbol /> Model must be less than 200 characters.</p> : null}
+                    {!valid('Make') ? <p> <ErrorSymbol />Make requires configuration to be setup with valuelists.</p> : null}
```

Comment:
just say "Make is required" I think this will confuse people who don't have access to ValuLists.